WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeCampaignCreationFormViewModelTests.swift

```diff
@@ -137,12 +137,15 @@ final class BlazeCampaignCreationFormViewModelTests: XCTestCase {
                                                            storage: storageManager,
                                                            productImageLoader: imageLoader,
                                                            onCompletion: {})
-        // Load suggestions and ensure that ad cannot be edited
-        await viewModel.loadAISuggestions()
-        XCTAssertFalse(viewModel.canEditAd)
 
         // When
-        await viewModel.downloadProductImage()
+        await viewModel.loadAISuggestions()
+        XCTAssertTrue(viewModel.canEditAd)
+
+        // Start downloading the product image without waiting for it to finish.
+        Task {
+            await viewModel.downloadProductImage()
```

Comment:
As this method call is wrapped inside a separate `Task`, the image download might not start when we check for `XCTAssertTrue(viewModel.canEditAd)` in line 151. So, this unit test is not always reliably testing for the intended behaviour. 

I think we should remove this unit test as we are testing for code that doesn't exist anymore.

Reply:
Nice catch, I missed this 🤦‍♀️